src/goto-programs/vcd_goto_trace.cpp

```diff
@@ -13,9 +13,9 @@ Date: June 2011
 
 #include "vcd_goto_trace.h"
 
-#include <cassert>
 #include <ctime>
 
+#include <util/invariant.h>
```

Comment:
No invariant added?

Reply:
Removed as now redundant, invariant check is inside `to_union_expr`.